vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java

```diff
@@ -91,10 +91,9 @@ private void tryToWorkaroundException(WebElement element, Optional<Action> defau
         {
             try
             {
-                if (webDriverManager.isBrowserAnyOf(Browser.CHROME)
-                        && message.contains(". Other element would receive the click"))
+                if (message.contains(". Other element would receive the click"))
                 {
-                    javascriptActions.click(element);
+                    softAssert.recordFailedAssertion(COULD_NOT_CLICK_ERROR_MESSAGE + webDriverException);
```

Comment:
Why are you appending an exception? What is the resulting error?
org.vividus.softassert.SoftAssert#recordFailedAssertion(java.lang.String, java.lang.Throwable)

Reply:
> What is the resulting error?
![interException](https://user-images.githubusercontent.com/58571912/216054012-61535a35-7034-4625-9c36-9eee08a8c699.png)
> Why are you appending an exception?
It has pretty enough information about the error
> org.vividus.softassert.SoftAssert#recordFailedAssertion(java.lang.String, java.lang.Throwable)
I think previous way (see previous pic) in the result a little readable
![image](https://user-images.githubusercontent.com/58571912/216055076-383a255b-60fd-4cc9-9366-22b73caeeee4.png)

Reply:
IMHO you should record the exception with stacktrace

Reply:
for this case that's doubtful, stack trace won't give any useful information

Reply:
In any case, we can expand error in Allure for stack trace
![image](https://user-images.githubusercontent.com/58571912/216060494-3fabfb0e-fa90-463c-b200-08d8ff66b957.png)

Reply:
Since you all agree on the matter, it is up to you how to deal with the stacktrace. However, that doesn't explain why you are appending an exception instead of its message

Reply:
Yep, is it useful for the user to have the class name in error?